ops/docker-compose.yml

```diff
@@ -136,9 +136,9 @@ services:
     image: ethereumoptimism/message-relayer:${DOCKER_TAG_MESSAGE_RELAYER:-latest}
     entrypoint: ./relayer.sh
     environment:
-      MESSAGE_RELAYER__L1RPCPROVIDER: http://l1_chain:8545
-      MESSAGE_RELAYER__L2RPCPROVIDER: http://l2geth:8545
-      MESSAGE_RELAYER__L1WALLET: '0xdbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97'
+      MESSAGE_RELAYER__L_1_RPC_PROVIDER: http://l1_chain:8545
```

Comment:
Hmmm I don't love the `L_1` split. Could be worth processing the strings to either:
1. Combine `_L_1` into `_L1` and same for L2 or
2. Combine *any* set of numbers into the previous set of characters

I think the L1/L2 thing might be good enough, tbh.